docs/README.openSUSE.md

```diff
@@ -72,6 +72,8 @@ sudo zypper ref
 
 If you get a `package not found` type of message with the below command, remove the offending package(s) from the install list and reissue the command. Take a note of the missing dependencies and, after a successful step completion, **[build the missing dependencies manually](#31-build-missing-dependencies)**.
 
+**NOTE:** Kodi requires a compiler with C++14 support, i.e. gcc >= 4.9 or clang >= 3.4
+
```

Comment:
Not related to here, but must not also for Mac updated https://github.com/xbmc/xbmc/blame/master/docs/README.macOS.md#L61 ? Not so much experience with xcode, but I think Xcode 8.x can't do it.

Reply:
Xcode 8.0 uses clang 3.9.0svn, https://en.wikipedia.org/wiki/Xcode#Xcode_7.0_-_11.x_(since_Free_On-Device_Development), so everything is fine